src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumChest.java

```diff
@@ -134,6 +135,25 @@ protected void addDisplayInformation(List<ITextComponent> textList) {
     public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, boolean advanced) {
         super.addInformation(stack, player, tooltip, advanced);
         tooltip.add(I18n.format("gregtech.machine.quantum_chest.capacity", maxStoredItems));
+
+        NBTTagCompound compound = stack.getTagCompound();
+        if (compound != null) {
+            String translationKey = null;
+            long count = 0;
+            if (compound.hasKey("ItemStack")) {
+                translationKey = new ItemStack(compound.getCompoundTag("ItemStack")).getUnlocalizedName() + ".name";
+                count = compound.getLong("ItemAmount");
+            } else if (compound.hasKey("PartialStack")) {
+                ItemStack tempStack = new ItemStack(compound.getCompoundTag("PartialStack"));
+                translationKey = tempStack.getUnlocalizedName() + ".name";
+                count = tempStack.getCount();
+            }
+            if (translationKey != null) {
+                tooltip.add(I18n.format("gregtech.machine.quantum_chest.tooltip.item",
+                        I18n.format(translationKey)));
+                tooltip.add(I18n.format("gregtech.machine.quantum_chest.tooltip.count", count));
```

Comment:
Tooltip regarding cleaning content by placing in crafting grid would be nice too.